contracts/Tokens/VTokens/VToken.sol

```diff
@@ -508,13 +509,12 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         accrualBlockNumber = currentBlockNumber;
         borrowIndex = borrowIndexNew;
         totalBorrows = totalBorrowsNew;
-        totalReserves = totalReservesNew;
```

Comment:
Why are we removing this? If we don't do this we won't increase the reserves every time we accrue interests

Reply:
Because the updated totalReservesValue is also set in `_ReduceReservesFresh` it seemed logical to calculate it in one place